docker-api/src/main/java/com/yahoo/vespa/hosted/dockerapi/Container.java

```diff
@@ -9,18 +9,21 @@
 public class Container {
     public final String hostname;
     public final DockerImage image;
+    public final ContainerResources flavor;
     public final ContainerName name;
     public final State state;
     public final int pid;
 
     public Container(
             final String hostname,
             final DockerImage image,
+            final ContainerResources flavor,
```

Comment:
Are you sure you want to call it flavor? Could be confusing since we already have something else called that. How about 'resources'?

Reply:
Oh! I initially named the class `ContainerFlavor`, but then renamed and forgot about the variables. Thanks.